### ISSUE TITLE
Issue with Credentials file when downloading project 

### ISSUE THREAD

**Issue by trishamoyer** [IBM/watson-banking-chatbot#83]
For a small error, on step 5 of the knowledge guide, it states "Rename the .env.example file to be .env and update it with valid credentials."
When downloading the zip, there is a file called env.example (missing the . at the beginning)

More important error here:
The following credentials are asked when replacing the environment variables:
However, these credentials don't exist

```
# Replace the credentials here with your own.
# Rename this file to .env before starting the app.

# Watson conversation
CONVERSATION_USERNAME=<add_conversation_username>
CONVERSATION_PASSWORD=<add_conversation_password>
WORKSPACE_ID=<add_conversation_workspace>

# Watson Discovery
DISCOVERY_USERNAME=<add_discovery_username>
DISCOVERY_PASSWORD=<add_discovery_password>
DISCOVERY_ENVIRONMENT_ID=<add_discovery_environment>
DISCOVERY_COLLECTION_ID=<add_discovery_collection>

# Watson Natural Language Understanding
NATURAL_LANGUAGE_UNDERSTANDING_USERNAME=<add_nlu_username>
NATURAL_LANGUAGE_UNDERSTANDING_PASSWORD=<add_nlu_password>

# Watson Tone Analyzer
TONE_ANALYZER_USERNAME=<add_tone_analyzer_username>
TONE_ANALYZER_PASSWORD=<add_tone_analyzer_password>

# Run locally on a non-default port (default is 3000)
# PORT=3000
```

**Comment by markstur** [comment]
Hopefully the more important creds don't exist is fixed with the IAM support in #82 

The knowledge guide part would still need to be addressed so I want to keep this open.

**Comment by markstur** [comment]
@dolph knowledge guide tweak help?
